patches/chrome-browser-profiles-profile_attributes_entry.cc.patch

```diff
@@ -1,12 +1,18 @@
 diff --git a/chrome/browser/profiles/profile_attributes_entry.cc b/chrome/browser/profiles/profile_attributes_entry.cc
-index ddbb9f9b9e44d5e8f009bf369e22ce67298d9670..a63b0e4834f590b78cdf8638f428b182f1e3801d 100644
+index ddbb9f9b9e44d5e8f009bf369e22ce67298d9670..eff80f5ba27257e52fe8d7ab2f588812c5f031cf 100644
 --- a/chrome/browser/profiles/profile_attributes_entry.cc
 +++ b/chrome/browser/profiles/profile_attributes_entry.cc
-@@ -1102,6 +1102,7 @@ bool ProfileAttributesEntry::ClearValue(const char* key) {
+@@ -1102,11 +1102,13 @@ bool ProfileAttributesEntry::ClearValue(const char* key) {
  
  // This method should be periodically pruned of year+ old migrations.
  void ProfileAttributesEntry::MigrateObsoleteProfileAttributes() {
 +  BRAVE_PROFILE_ATTRIBUTES_ENTRY_MIGRATE_OBSOLETE_PROFILE_ATTRIBUTES
    // Added 3/2023.
    ClearValue(kAccountCategories);
  }
+ 
+ void ProfileAttributesEntry::SetIsOmittedInternal(bool is_omitted) {
++  BRAVE_SET_IS_OMITTED_INTERNAL
```

Comment:
why are we patching this instead of just calling it when creating the profile?

Reply:
1) I am (was) calling it when creating the profile, but chromium's SetIsOmittedInternal CHECKs if it's not an ephemeral profile. 
The explanation is where the macro is defined:
```
// Allow non-ephemeral profiles to be omitted for the purposes of
// built-in profiles like the AI Chat Agentic Profile.
```

2) I failed to realize that this isn't persisted anyway, so there's no point to setting it (and the override to allow it for non-ephemeral profiles), so I've removed this part